platform/commonUI/edit/src/creation/CreateAction.js

```diff
@@ -86,11 +86,23 @@ define(
                         })
                         .join('/');
 
-                openmct.router.navigate(url);
+                function editObject() {
+                    const path = objectPath.slice(-1).map(obj => {
+                        const objNew = obj.getCapability('adapter');
+                        objNew.identifier = openmct.objects.parseKeyString(obj.getId());
```

Comment:
The adapter capability doesn't automatically give them an identifier? It seems like it maybe should?

Reply:
No it doesn't. :( 
Do you want me to add that? I didn't want to touch legacy code at this point since it will go away soon anyway.

Reply:
Hrm, this shouldn't be necessary. The adapter capability ultimately calls the `toNewFormat` function in object-utils [which assigns an identifier](https://github.com/nasa/openmct/blob/master/src/api/objects/object-utils.js#L137).